src/scenes/Lobby/Lobby.tsx

```diff
@@ -1,19 +1,73 @@
-import { Link } from 'react-router-dom';
-import logo from '../../assets/dummy/lobby-logo.png';
+import { useState } from 'react';
+import { CheckCircle, Copy } from 'react-feather';
+import Background from '../../components/Background';
+import Header from '../../components/Header';
+import Button from '../../components/Button';
+import PlayerList from './PlayerList';
 import './Lobby.css';
 
 function Lobby() {
+  const userData = JSON.parse(window.localStorage.getItem('userData'));
```

Comment:
Precisamos mesmo pegar esses dados toda vez que o componente renderizar?
Me parece fazer sentido colocar isso dentro de um useEffect e pegar esses dados só quando o componente for montado.

Reply:
done!